compair/static/modules/assignment/assignment-view-partial.html

```diff
@@ -148,7 +148,7 @@ <h2>All answers
 
                 <!-- Students should see before evaluating during the comparison period (regular comparison) -->
                 <p class="intro-text" ng-if="!canManageAssignment && assignment.compare_period && comparisons_left > 0 && ((assignment.status.answers.answered && assignment.answer_period) || !assignment.answer_period)">
-                    <a ng-href="#/course/{{courseId}}/assignment/{{assignment.id}}/compare" title="Compare answers">Compare answers</a> for the assignment first, then read answers written by your classmates and instructors here.
+                    <a ng-if="hasComparisonsAvailable" ng-href="#/course/{{courseId}}/assignment/{{assignment.id}}/compare" title="Compare answers">Compare answers</a><span ng-if="!hasComparisonsAvailable">Compare answers (when they are available)</span> for the assignment first, then read answers written by your classmates and instructors here.
```

Comment:
`hasComparisonsAvailable` seems not defined here.  The link isn't showing even when there are available comparisons.  Maybe use `assignment.status.comparisons.available` instead?

Reply:
Ha, yep, committed wrong version. Thanks for the catch.